examples/examples.js

```diff
@@ -43,6 +43,7 @@ var categories = [
             "portal",
             "post-effects",
             "render-to-texture",
+            "refraction",
```

Comment:
So you can achieve refraction with a regular standard material using a box-projected cubemap. This is maybe better named as 'dynamic-refraction' or 'grab-pass' IMHO. And if I picked on of those, I'd go with 'grab-pass'.

Reply:
done, cheers.